assets/charts/control-plane/kubelet/container-image/README.md

```diff
@@ -0,0 +1,57 @@
+# Lokomotive Kubelet Image
+
+This image is built on top of the Typhoon Kubelet image. This image includes a wrapper script that runs `isciadm` in the host namespace. This ensures that all the `isciadm` libraries are loaded from the host.
+
+## Updating Image Tag
+
+In this directory run the following command to update the Kubernetes version:
+
+```bash
+export CURRENT_VERSION=
+export NEW_VERSION=
+```
+
+```bash
+sed -i "s|$CURRENT_VERSION|$NEW_VERSION|g" Dockerfile README.md
+```
+
+## Building Image
+
+### x86
+
+```bash
+ARCH=amd64 docker build -t quay.io/kinvolk/kubelet:v1.19.4-amd64 --build-arg ARCH . \
```

Comment:
Can't we use env variables in those commands? Then we don't have to replace anything. Just set one variable at the beginning. It would make everything much simpler.